docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/rdb-loader-configuration-reference/index.md

```diff
@@ -27,11 +27,14 @@ This is a complete list of the options that can be configured:
 | `host`                     | Required. Host name of Redshift cluster.                                                                                                                                        |
 | `port`                     | Required. Port of Redshift cluster.                                                                                                                                             |
 | `database`                 | Required. Redshift database which the data will be loaded to.                                                                                                                   |
-| `roleArn`                  | Required. AWS Role ARN allowing Redshift to load data from S3.                                                                                                                  |
+| `roleArn`                  | Required if 'NoCreds' is chosen as load auth method.. AWS Role ARN allowing Redshift to load data from S3.                                                                                                                  |
 | `schema`                   | Required. Redshift schema name, eg “atomic”.                                                                                                                                    |
 | `username`                 | Required. DB user with permissions to load data.                                                                                                                                |
 | `password`                 | Required. Password of DB user.                                                                                                                                                  |
 | `maxError`                 | Optional. Configures the [Redshift MAXERROR load option](https://docs.aws.amazon.com/redshift/latest/dg/copy-parameters-data-load.html#copy-maxerror). The default is 10.       |
+| `loadAuthMethod.*`         | Optional, default method is `NoCreds`. Specifies the auth method to use with 'COPY' statement. |
```

Comment:
```suggestion
| `loadAuthMethod.*` (since 5.2.0) | Optional, default method is `NoCreds`. Specifies the auth method to use with the `COPY` statement. |
```